services/qr/handler.js

```diff
@@ -62,36 +62,29 @@ export const post = async (event, ctx, callback) => {
 
     await registrationHelpers
       .qrScanPostHelper(data, data.email)
-      .then((res) => {
+      .then(async (res) => {
         if (res.hasOwnProperty("errorMessage")) {
-          if (
-            res.errorMessage === "Team scan would result in negative points"
-          ) {
-            const response_fail = helpers.createResponse(406, {
-              message: "ERROR: " + res.errorMessage,
-              response: res
-            });
-
-            callback(null, response_fail);
-            return response_fail;
-          }
-
           const response_fail = helpers.createResponse(403, {
             message: "ERROR: " + res.errorMessage,
             response: res
           });
-
           callback(null, response_fail);
           return response_fail;
+        } else {
+          try {
+            await registrationHelpers.logQRScan(data.qrCodeID, data.email);
+            console.log("hey success!!");
```

Comment:
before submitting PRs, a good practice is to look at every line that you changed to make sure each line change is intended. 

Let's make these console logs more meaningful

Reply:
ok i removed the debugging console logs